.gitignore

```diff
@@ -23,3 +23,6 @@ bin/main/application.yaml
 
 applicationinsights-agent-*.jar
 *.log
+
+### Visual Studio Code ###
+bin
```

Comment:
Can you remove the files in `bin/` that have been added as part of this PR

Reply:
done

Reply:
Oh it looks like `bin/run-in-docker.sh` shouldn't have been removed?

Might need this

```
bin
!bin/run-in-docker.sh
```

Reply:
added back in now. Couldn't exclude it like you suggested, as according to git:-

> It is not possible to re-include a file if a parent directory of that file is excluded. Git doesn’t list excluded directories for performance reasons, so any patterns on contained files have no effect, no matter where they are defined.

https://git-scm.com/docs/gitignore

So have specified the folders within bin to exclude individually.

Reply:
although it comes up as an empty file in the diff view, but displays fine if you look at the branch, which is odd.